File: backend/src/domain/entities/FundReceivingInfo.ts
import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';

@Entity('fund_receiving_infos')
export class FundReceivingInfo {
  @PrimaryGeneratedColumn()
  id: number;

  @Column({ name: 'fund_id' })
  fundId: number;

  @Column({ type: 'varchar', nullable: true })
  type: string;

  @Column({ type: 'varchar', name: 'account_number', nullable: true })
  accountNumber: string;

  @Column({ type: 'varchar', name: 'account_name', nullable: true })
  accountName: string;

  @Column({ type: 'varchar', name: 'bank_name', nullable: true })
  bankName: string;

  @Column({ type: 'varchar', name: 'qr_code_url', nullable: true })
  qrCodeUrl: string;

  @Column({ type: 'boolean', name: 'is_active', nullable: true })
  isActive: boolean;
}

File: backend_java/src/main/java/com/service/backend/domain/entity/FundReceivingInfo.java
package com.service.backend.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table("fund_receiving_infos")
public class FundReceivingInfo {

    @Id
    private Long id;

    @Column("fund_id")
    private Long fundId;

    private String type;

    @Column("account_number")
    private String accountNumber;

    @Column("account_name")
    private String accountName;

    @Column("bank_name")
    private String bankName;

    @Column("qr_code_url")
    private String qrCodeUrl;

    @Column("is_active")
    private Boolean isActive;
}
